src/fileformat/types/import_table/import_table.cpp

```diff
@@ -675,6 +675,14 @@ const std::string& ImportTable::getImphashSha256() const
 	return impHashSha256;
 }
 
+/**
+ * Get imphash as tlsh
+ * @return Imphash as tlsh
+ */
```

Comment:
You don't have to write comments like these. Ideally functions should be clear from their signature - name, return type, argument types and names. This one is. The comment is not adding any useful new information.